.github/workflows/ci.yaml

```diff
@@ -7,12 +7,7 @@ concurrency:
   cancel-in-progress: true
 
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-  schedule:
-    - cron: '53 0 * * *' # Daily at 00:53 UTC
```

Comment:
Why are you deleting nightly tests and CI tests on PR creation?

Reply:
For PRs, there is an existing flow ```on_pull_request.yaml``` which calls this ```ci.yaml```. So, we are covered there.

For the nightly runs, let me add that back.

Reply:
nightly runs and on PR directive was added back as Alex requested.